eventbridge-fan-in-terraform/README.md

```diff
@@ -0,0 +1,60 @@
+# Amazon Eventbridge Eventbus fan-in to Central Eventbus to different region
+
+This pattern demonstrates how to aggregate all your events from multiple Eventbus (in the same region) to a central Eventbus in a different region. This allows you to centrally accumulate events coming in from different event sources for downstream consumption.
+
+This pattern is deployed using Terraform to create a central EventBridge bus, Eventbridge rules on fan-in buses and all IAM resources required. The Eventbuses to aggregate can be defined in terraform.tfvars file (Sample ARNs is provided, replace with Eventbus ARNs as needed). The provider.tf file also lists the AWS regions of the fan-in Eventbus and central Eventbus (replace these based on where your Eventbuses exist and where you want your central bus to be created).
+
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/eventbridge-fan-in-terraform
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/aws-get-started) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd eventbridge-fan-in-terraform
+    ```
+3. In the provider.tf file, add the regions where the Eventbuses exist in the provider block with alias "others". Also add the region where you want the central Eventbus to be created in the provider block with alias "central"
+
+4. In the terraform.tfvars file, add the ARNs of the Eventbuses you want to aggregate. Note that these Eventbus should exist in the same region
+
+5. From the command line, initialize Terraform:
+    ```
+    terraform init
+    ```
+6. From the command line, apply the configuration in the main.tf file and follow the prompts:
+     ```
+    terraform apply
```

Comment:
Can you please add some additional info as to what value is expected here.
I see the input value is of type `list(string)`
I think this is to create additional event busses.
If I enter `["other1"]` which I think matches the format I get an error:
```
│ Error: creating EventBridge Rule (terraform-20230908144448235100000001): ResourceNotFoundException: Event bus other1 does not exist.
│
│   with aws_cloudwatch_event_rule.ebrule["other1"],
│   on main.tf line 6, in resource "aws_cloudwatch_event_rule" "ebrule":
│    6: resource "aws_cloudwatch_event_rule" "ebrule" {
```

Reply:
Sorry about that, I think that was because the terraform.tfvars file was not committed to the repo. Updated the new commit. Step 4 of the Readme specifies what values are expected by terraform.tfvars.
Can you please try it again?

Reply:
Do you need to manually pre-create event buses in the fan-in regions for this to work? 
Can I suggest to be more explicit about what is required to test this pattern.
To test this pattern create x event busses in x regions.  (is there a minimum to get this to work, can I just create 1 additional?)
Then specify to Enter their arns in `terraform.tfvars`

Reply:
Apologies for the unclear instructions. I have updated the Readme.md file to add more details on how many existing Eventbuses need to exist and disclaimer that the pattern assumes the fan-in buses already exist in the same region. Please let me know if any more details are needed